Formula/aspectj.rb

```diff
@@ -25,7 +25,7 @@ class Aspectj < Formula
 
   def install
     mkdir_p "#{libexec}/#{name}"
-    system "#{Formula["openjdk"].bin}/java", "-jar", "aspectj-#{version}.jar", "-to", "#{libexec}/#{name}"
+    system "#{Formula["openjdk"].bin}/java", "-jar", "installer-#{version}.jar", "-to", "#{libexec}/#{name}"
```

Comment:
raised this artifact naming change question, https://github.com/eclipse/org.aspectj/issues/202